app/views/widgets/_smart_banner.html.slim

```diff
@@ -0,0 +1,13 @@
+- cookie = cookies[:smart_banner_dismiss]
+- if (browser.device.mobile? || browser.device.tablet?) && (cookie.nil? || (Time.now >= Time.strptime(cookie, "%FT%R") + 15.days))
+  .smart-banner
+    .info
+      i.icon-close
+      a href="#{browser.platform.android? ? android_store_page : apple_store_page}"
+        = image_tag "icon", class: "app-icon"
+      h3 Baixe o app da Mudamos
+  javascript:
+    $(".smart-banner .info i.icon-close").click(function() {
```

Comment:
Protip: avoid over specification of markup on selectors. If you happen to include a element on the hierarchy, your selector would break. In this case, just `$(".smart-banner .icon-close")` would be enough.